certified-connectors/IA-Connect Session/apiDefinition.swagger.json

```diff
@@ -4173,6 +4185,658 @@
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "CredentialAddress",
+                "CredentialType",
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "UserName": {
+                  "type": "string",
+                  "title": "User name",
+                  "description": "The credential user name."
+                },
+                "Password": {
+                  "type": "string",
+                  "title": "Password",
+                  "description": "The credential password."
+                }
+              }
+            }
+          }
+        },
+        "operationId": "CredentialRead"
+      }
+    },
+    "/Environment/CredentialDelete": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Deletes a user credential",
+        "description": "Deletes a credential in the user's profile in the IA-Connect Agent session. Credentials can be viewed from the Windows Control Panel.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "CredentialDelete",
+            "description": "Inputs for IA-Connect CredentialDelete action",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "CredentialAddress": {
+                  "type": "string",
+                  "title": "Credential address",
+                  "description": "The name / address of the credential."
+                },
+                "CredentialType": {
+                  "type": "string",
+                  "title": "Credential type",
+                  "description": "The credential type: Windows or Generic.",
+                  "enum": [
+                    "Windows",
+                    "Generic"
+                  ]
+                },
+                "Workflow": {
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "CredentialAddress",
+                "CredentialType",
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}

```

Comment:
@LukeMcGovern-UL Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
@sriyen-msft Thank you for reviewing our pull request - in this scenario the command returns no output and therefore has no output schema. We distinguish between success or failure by returning the relevant HTML code, 200 (success) or 400 (failure) which will result in an exception in the Power Automate Flow. 

Most of our actions do return data and all of these have an appropriate output schema defined.